### PR TITLE
WIP: Ember 1.10 Compatibility

### PR DESCRIPTION
**Pull request by taras**
**Done:**

- Added compatibility with Ember 1.10
- Wrapped a few more view.create() in run loop

**TODO:**

- [ ] Test in Ember 1.9
- [ ] Figure out why tests fail in Chrome (but pass in Phantom)

1__ember_test_--server__phantomjs__and_main_window

### CODE REVIEW

**Comment by ebryn** [comment]
What's the latest here?

**Comment by gigafied** [comment]
Just tried using this fork with Ember 1.10.0, I get "Uncaught Error: Assertion Failed: A fragment cannot be pushed into a buffer that contains content", which I suspect is due to how I'm using it. I have a `ContainerView` and am using `createChildView` to create the list-view instance.

File: packages/list-view/lib/helper.js [hunk 1]
```diff
@@ -6,20 +6,23 @@ var EmberList = createHelper(EmberListView);
...
+            hash[camelized] = hash[prop];
+            delete hash[prop];
+          }
         }
-      }
 
-      /*jshint validthis:true */
-      return Ember.HTMLBars.helpers.collection.helperFunction.call(this, [view], hash, options, env);
+        /*jshint validthis:true */
+        return Ember.Handlebars.helpers.collection.helperFunction.call(this, [view], hash, options, env);
```
Comment (jasonmit) [inline packages/list-view/lib/helper.js]: `env.helpers.collection.helperFunction`

**Comment by zachaysan** [comment]
Hey when is this getting merged? :8ball:

**Comment by alexdiliberto** [comment]
Has this been abandoned in favor of the full [CLI Addon integration PR](https://github.com/emberjs/list-view/pull/203)?

**Comment by jasonmit** [comment]
Ping @taras

**Pull request merged**